src/renderer/kube-object.store.ts

```diff
@@ -11,7 +11,8 @@ import { getHostedCluster } from "../common/cluster-store";
 @autobind()
 export abstract class KubeObjectStore<T extends KubeObject = any> extends ItemStore<T> {
   abstract api: KubeApi<T>;
-  public limit: number;
+  public readonly abstract limit: number;
+  public readonly abstract saveLimit: number;
```

Comment:
Same here, probably cannot be abstract. Also `bufferSize` might be a better name?